talk/setup.tex

```diff
@@ -76,6 +76,7 @@
 
 \usepackage{booktabs}
 \usepackage{ifthen}
+\usepackage{epsdice}
```

Comment:
Drop, as you're not using it. On top I'm not sure it's worth depending on such a package